CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # Changelog
 
+## 0.0.3 - 2023-05-04
+
+### Features
+
+- **Individual Changelog pages: [shadows-withal]/[↬284]**
+    
+    When announcing a new release- it's often desirable to link to an individual
+    release page that contains the changelog/release notes. Previously, we built
+    a single page for all the releases- now we build individual pages as well.
+
+    This is the first shipped feature from our new team member, Liv! Yay and 
+    welcome :)
+
+### Fixes
+
+- **Improved configuration support for non-cargo dist users - [pomdtr]/[#262] , [ashleygwilliams]/[↬281]**
+
+    Previously, setting `cargo-dist` as false, or omitting it should have been
+    sufficient to stop oranda from attempting to parse your releases as
+    cargo-dist artifacts, however `cargo-dist: false` did not work! How the
+    entire artifacts config object is handled has been reworked and tested.
+    Addiitionally, work from a refactor has also made project with a mixture of
+    cargo-dist and non-cargo-dist releases work much better.
+
+- **Installer header fallback when platform is not detected: [gankra]/[↬269]**
+
+    When you use cargo-dist, we display a header on your index page that
+    detects your user's platform and recommends and installer to them. We did
+    not previously have a fallback if we detected a system that none of the
+    installers supported. Now, in that scenario- we'll offer an artifact
+    download as the header option.
+
+- **Dev commands has proper default values: [ashleygwilliams]/[#256],[↬260]**
+
+    Due to a false hope that `#[derive(Default)]` would collect default from 
+    the `clap` derive API, we shipped the `dev` command with null defaults.
+    `dev` now has the same defaults that `serve` does, as is to be expected.
+
+- **Improved typogrphy and layout styles on mobile = [ashleygwilliams]/[#234], [SaraVieira]/[↬276]**
+
+### Maintenance
+
+- **Data fetching refactor: [ashleygwilliams]/[#226],[↬274]**
+- **Update actions/checkout to v3: [ashleygwilliams]/[#251],[↬255]**
+
+[#209]: https://github.com/axodotdev/oranda/issues/209
```

Comment:
(nag) why are some of these with the `#` symbol and some of them with the `↬` symbol?

Reply:
some are issues and some a PRs ;) - i do think because i wanna celebrate when an external contributor files a good issue

Reply:
ah okay, fair enough, shakes hands at github for not differentiating there

Reply:
yeah usually i will write out issue or pr or something else but a previous colleague used the little arrow and i kinda liked it- it is true it's not exactly obvious tho